src/melee.cpp

```diff
@@ -471,6 +471,16 @@ bool Character::melee_attack_abstract( Creature &t, bool allow_special,
                                        const matec_id &force_technique,
                                        bool allow_unarmed )
 {
+    if( weapon.has_flag( flag_CRUTCHES ) && get_working_leg_count() < 2 ) {
+        if( !movement_mode_is( move_mode_id( "prone" ) ) ) {
+            add_msg_if_player( "Your broken legs cannot hold you and you fall down" );
+            set_movement_mode( move_mode_id( "prone" ) );
+        } else if( is_on_ground() ) {
+            add_msg_if_player( "You cannot fight while on the ground" );
```

Comment:
You can still bash zombies' legs while on the ground, so I think it's fine to allow attacking in this case.

On the other hand, running or crouching should be forbidden when the legs are broken and crutches are used, if it's not already so.